pkg/k8s/apis/cilium.io/v1alpha1/tracing_policy_types.go

```diff
@@ -90,6 +90,13 @@ type TracingPolicySpec struct {
 	// PodSelector selects pods that this policy applies to
 	PodSelector *slimv1.LabelSelector `json:"podSelector,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// ContainerSelector selects containers that this policy applies to.
+	// A map of container fields will be constructed in the same way as a map of labels.
+	// The name of the field represents the label "key", and the value of the field - label "value".
+	// Currently, only the "name" field is supported.
+	ContainerSelector *slimv1.LabelSelector `json:"containerSelector,omitempty"`
```

Comment:
I was confused at first by `LabelSelector`, as there are no labels involved, but I guess by doing so we get `MatchExpressions` in addition to exact name match essentially for free? This seems like a good enough reason to reuse `LabelSelector`, just checking my understanding.

Reply:
Thank you for the review!
> I see there is a conflict in rthooks, hopefully not painful. Could you rebase and resolve it?

Sure, it was caused by the addition of the error label to the `policyfilter_metrics_total` label. I resolved it:)

------
> I was confused at first by LabelSelector, as there are no labels involved, but I guess by reusing it we get MatchExpressions in addition to exact name match essentially for free?

You are totally right! I imagined container fields the same as labels (a map of key-value pairs) and therefore decided to reuse the existing filtering logic. As you said, essentially for free :sweat_smile: